src/wallet/rpcwallet.cpp

```diff
@@ -1835,61 +1441,40 @@ static void ListTransactions(CWallet* const pwallet, const CWalletTx& wtx, const
     {
         for (const COutputEntry& r : listReceived)
         {
-            std::string account;
+            std::string label;
             if (pwallet->mapAddressBook.count(r.destination)) {
-                account = pwallet->mapAddressBook[r.destination].name;
+                label = pwallet->mapAddressBook[r.destination].name;
             }
-            if (fAllAccounts || (account == strAccount))
+            UniValue entry(UniValue::VOBJ);
+            if (involvesWatchonly || (::IsMine(*pwallet, r.destination) & ISMINE_WATCH_ONLY)) {
+                entry.pushKV("involvesWatchonly", true);
+            }
+            MaybePushAddress(entry, r.destination);
+            if (wtx.IsCoinBase())
             {
-                UniValue entry(UniValue::VOBJ);
-                if (involvesWatchonly || (::IsMine(*pwallet, r.destination) & ISMINE_WATCH_ONLY)) {
-                    entry.pushKV("involvesWatchonly", true);
-                }
-                if (IsDeprecatedRPCEnabled("accounts")) entry.pushKV("account", account);
-                MaybePushAddress(entry, r.destination);
-                if (wtx.IsCoinBase())
-                {
-                    if (wtx.GetDepthInMainChain() < 1)
-                        entry.pushKV("category", "orphan");
-                    else if (wtx.IsImmatureCoinBase())
-                        entry.pushKV("category", "immature");
-                    else
-                        entry.pushKV("category", "generate");
-                }
+                if (wtx.GetDepthInMainChain() < 1)
+                    entry.pushKV("category", "orphan");
+                else if (wtx.GetBlocksToMaturity() > 0)
```

Comment:
Any reason for this change?

Reply:
No reason, except a bad rebase. Thanks for catching this!

Fixed in https://github.com/bitcoin/bitcoin/pull/14023/commits/bb08423d5ca866d4a139a3b57ff110d818d08b32

Reply:
Good catch Marco. Since this slipped past me, I did some research and found that the CLion IDE effectively highlights the change, while github and diff2html do not. Adding it to my review process.

<img width="513" alt="clion ide" src="https://user-images.githubusercontent.com/6299681/44679167-30ccf980-aa08-11e8-8561-3b335e2ea5b3.png">
![image](https://user-images.githubusercontent.com/6299681/44679344-b3ee4f80-aa08-11e8-840f-82f6684d8b26.png)